From 16842367048d0c61e6c84f7490589b6cfb8a0ad2 Mon Sep 17 00:00:00 2001
From: Fernando Fernandez Mancera <ffmancera@riseup.net>
Date: Thu, 21 Nov 2024 00:40:27 +0100
Subject: [PATCH] fix(35network-manager): install nft binary during module
 installation

NetworkManager has a new bonding mode called balance-slb. This mode is
used in environments where NICs are connected to switches without LACP.
In order to work, NetworkManager configures a set of nftables rules.

The 'nft' binary is required to work.

Signed-off-by: Fernando Fernandez Mancera <ffmancera@riseup.net>

(cherry picked from commit 37317b61fda5181aeb0604381e994e8181a53200)

Resolves: RHEL-69461
---
 modules.d/35network-manager/module-setup.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/35network-manager/module-setup.sh b/modules.d/35network-manager/module-setup.sh
index 2c1fe718..80ca21bd 100755
--- a/modules.d/35network-manager/module-setup.sh
+++ b/modules.d/35network-manager/module-setup.sh
@@ -32,7 +32,7 @@ install() {
     inst NetworkManager
     inst_multiple -o /usr/{lib,libexec}/nm-initrd-generator
     inst_multiple -o /usr/{lib,libexec}/nm-daemon-helper
-    inst_multiple -o teamd dhclient
+    inst_multiple -o teamd dhclient nft
     inst_hook cmdline 99 "$moddir/nm-config.sh"
     if dracut_module_included "systemd"; then
 

